launch_testing/launch_testing/loader.py

```diff
@@ -48,6 +49,16 @@ def __init__(self,
         self.pre_shutdown_tests = pre_shutdown_tests
         self.post_shutdown_tests = post_shutdown_tests
 
+        # If we're parametrized, extend the test names so we can tell more easily what
+        # params they were run with
+        if self.param_args:
+            for tc in itertools.chain(_iterate_tests_in_test_suite(pre_shutdown_tests),
+                                      _iterate_tests_in_test_suite(post_shutdown_tests)):
```

Comment:
Commonly we try to avoid this kind of alignment and wrap after the open parenthesis. I just took the liberty to apply that change to get this merged quickly: 7baa83e2bb1e4d3c9d5df60f6ede09206b6da7e5